INSTALL.md

```diff
@@ -28,7 +27,7 @@ DONATE_SECRET="<random bits of entropy, available in the woodshop>"
 ```
 
 # Database installation
-`$ flask db init` will complain if the migrations directory exists, which it will since it's included in the package.  It's fine.  When it does complain, make sure the directory migrations/versions exists.  If not, just create the directory.
+`$ flask db init` will complain if the migrations directory exists, which it will since it's included in the package.  It's fine.  When it does complain, make sure the directory migrations/versions exists.  If not, just create the directory, but migrations/verions is shipped as well.
```

Comment:
`If not, just create the directory, but migrations/verions is shipped as well.` seems a bit confusing to me but I don't have a good wording off the top of my head